lib/braintree.js

```diff
@@ -0,0 +1,11 @@
+import { loadScriptAsync } from './utils';
+
+export const getBraintree = async () => {
+  if (typeof window.braintree === 'undefined') {
+    // No need to await for data-collector
+    loadScriptAsync('https://js.braintreegateway.com/web/3.71.0/js/data-collector.min.js');
+    await loadScriptAsync('https://js.braintreegateway.com/web/dropin/1.26.0/js/dropin.min.js');
+  }
+
+  return window.braintree;
```

Comment:
Is that await enough to ensure this will never be undefined?

Reply:
If the script is loaded properly, this will normally always be set (unless their version in production is broken, or if they rename this variable). If there's a network error, an exception will be raised that we should handle in https://github.com/opencollective/opencollective-frontend/blob/04549c6bb4248c513b28185e2037d7cf67a12d81/components/contribution-flow/BraintreePaymentForm.js#L55 to display a proper message. I'm adding a card for that in https://github.com/opencollective/opencollective/projects/52